src/sql/workbench/browser/modelComponents/queryTextEditor.ts

```diff
@@ -58,7 +58,7 @@ export class QueryTextEditor extends BaseTextEditor {
 	protected getConfigurationOverrides(): IEditorOptions {
 		const options = super.getConfigurationOverrides();
 		if (this.input) {
-			options.inDiffEditor = true;
+			options.inDiffEditor = false;
```

Comment:
I'm trying to figure out why this was ever set to true but in https://github.com/microsoft/azuredatastudio/pull/2058 it was set that way and there weren't any comments about it in the PR that I can find.

Reply:
I've tested this and don't see anything immediately wrong here. Really weird why this was set this way to begin with.